.github/workflows/build-publish-containers.yml

```diff
@@ -32,8 +32,9 @@ jobs:
       contents: read
     strategy:
       matrix:
-        tag: ['client:centos7', 'build-node:debian11']
-        platform: [amd64, arm64, ppc64le]
+        #tag: ['client:centos7', 'build-node:debian11']
```

Comment:
That's just for testing purposes I guess?

Reply:
Yep :-)

Reply:
I changed it back, and it's building all images again now.